runtime-common/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@cardstack/runtime-common",
+  "version": "0.0.0",
+  "license": "MIT",
+  "main": "index.js"
+}
```

Comment:
I bit the bullet and made a `runtime-common` workspace. the exact same externals maps also needed to be consumed by the CardInspector's resolver, and I _really_ didn't want to maintain the same mappings separately in both the host and the worker